### DISCUSSION TITLE
Multiple pdf tickets on one page

### DISCUSSION THREAD

**Discussion by mohe2015** [pretix/pretix#1950] [Ideas]
For environmental reasons I think it would be great to have an option to print multiple tickets onto one page.
The most basic implementation would be to add a pageHeight to allow multiple tickets on one page if the pageHeight is a multiple of the templateHeight. This would mean that only vertical stacking is possible which I think would be fine.

**Comment by raphaelm** [comment]
Hmm, just adding a page height might be the least complex solution that has been proposed for this so far, I like it :)

It would be nice to be able to do something stylish with the "empty space" when only one ticket is bought, but this adds so much complexity that we discarded the idea so far.

**Comment by moritzlerch** [comment]
Any updates on this? I would find that feature very useful because my print dialog currently just rotates the ticket (whose width is set to 21cm) and puts it vertically on the page resulting in a very big ticket (now with a new width of A4 height). 

<details><summary>image of the print dialog</summary>

![grafik](https://user-images.githubusercontent.com/59474941/235298494-34fe3ce6-2f57-4953-b9ce-6d5da6348ba1.png)
</details>
